transport/src/test/java/io/netty/channel/pool/SimpleChannelPoolTest.java

```diff
@@ -242,4 +237,43 @@ public void initChannel(LocalChannel ch) throws Exception {
         channel2.close().syncUninterruptibly();
         group.shutdownGracefully();
     }
+
+    @Test
+    public void testBootstrap() {
+        // Checking for the actual bootstrap object doesn't make sense here, since the pool uses a copy with a modified
+        // channel handler.
+        assertNotNull(new SimpleChannelPool(new Bootstrap(), new CountingChannelPoolHandler()).bootstrap());
```

Comment:
@jchambers just for demonstrate correct usage please also close the pool in each test.